modules/ServerAPI/src/server/python/syncProjectsUsers/ld_entitlements.py

```diff
@@ -20,6 +20,10 @@
 
 
 def parse_options(argv):
+    groupsJSONFile = open(argv[1], "r")
```

Comment:
I think this will leave the file open for reading. I don't think it matters deeply, but it is better form to do something like this, which will close the file after the "with" block ends
with open(argv[1], "r") as groupsJSONFile:
    import_json = groupsJSONFile.read()
    argv[1] = import_json